platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/ConsensusEvent.java

```diff
@@ -37,5 +38,15 @@ public interface ConsensusEvent extends Event, ReachedConsensus {
      * @return a consensus transaction iterator
      */
     @NonNull
+    @Deprecated
     Iterator<ConsensusTransaction> consensusTransactionIterator();
+
+    /**
+     * Returns an iterator over the application events in this transaction, which have all reached consensus in {@link Bytes} format. Each
+     * invocation returns a new iterator over the same transactions. This method is thread safe.
+     *
+     * @return a consensus transaction iterator
+     */
+    @NonNull
+    Iterator<Bytes> transactionBytesIterator();
```

Comment:
we still need the `ConsensusTransaction` wrapper, not just bytes

Reply:
Addressed